integration/v7/isolated/service_command_test.go

```diff
@@ -171,17 +171,16 @@ var _ = Describe("service command", func() {
 				})
 
 				It("displays the bound apps", func() {
-					//Delay to reduce flakiness
-					time.Sleep(10 * time.Second)
-					session := helpers.CF(serviceCommand, serviceInstanceName, "-v")
-					Eventually(session).Should(Exit(0))
-
-					Expect(session).To(SatisfyAll(
-						Say(`Showing bound apps:\n`),
-						Say(`name\s+binding name\s+status\s+message\n`),
-						Say(`%s\s+%s\s+create succeeded\s*\n`, appName1, bindingName1),
-						Say(`%s\s+%s\s+create succeeded\s*\n`, appName2, bindingName2),
-					))
+					Eventually(func(g Gomega) {
+						session := helpers.CF(serviceCommand, serviceInstanceName, "-v")
```

Comment:
```suggestion
						session := helpers.CF(serviceCommand, serviceInstanceName, "-v").Wait()
```

Reply:
This will wait until the process exits before continuing

Reply:
Is there any advantage of this approach vs the one that is in the PR?

Reply:
Oh just mentioned below, using .Wait() on the command will have it exit before continuing, so it can process the input and try again as needed

Reply:
Stopped using the buffer